### PR TITLE
Cycles

### PR DESCRIPTION
**Pull request by ElenaLadutska**
Added only cycles tasks

### CODE REVIEW

File: cycles.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+//   console.log(i);
+//   document.open();
+//   document.write(`<p>${i}</p>`);
+//   document.close();
+// };
+
+//Cycles, while, for. Fifth task
+
+// for(i = 0; i <= 100; i++){
+//   if(i % 2 === 0){
```
Comment (kathrynowy) [inline cycles.js]: how can we write condition here without === 0?

File: cycles.js [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+//   n = n/2;
+//   num++;
+// }while(n >= 50);
+// console.log(`Amount of iterations = ${num}`);
+// console.log(`Last number = ${n}`);
+
+//or
+
+// while(n >= 50){
+//   n = n / 2;
```
Comment (kathrynowy) [inline cycles.js]: /=

**Review by kathrynowy — CHANGES_REQUESTED** [review]

File: cycles.js (hunk 2 above)
Comment (ElenaLadutska) [reply cycles.js]: fixed

File: cycles.js (hunk 1 above)
Comment (ElenaLadutska) [reply cycles.js]: i = 0; i <= 100; i += 2

Comment (ElenaLadutska) [reply cycles.js]: i added it in my js file

Comment (kathrynowy) [reply cycles.js]: rewrite without === 0

**Review by kathrynowy — CHANGES_REQUESTED** [review]

File: cycles.js (hunk 1 above)
Comment (ElenaLadutska) [reply cycles.js]: fixed

**Review by kathrynowy — APPROVED** [review]

**Pull request merged**